package/src/index.ts

```diff
@@ -50,6 +50,41 @@ export class CandleClient {
     this.candle.initialize(appUser);
   }
 
+  public presentTradeExecutionSheet(input: {
+    tradeQuote: TradeQuote;
+    presentationBackground?: PresentationBackground;
+    completion?: (
+      result: ({ kind: "success" } & Trade) | { kind: "failure"; error: string }
+    ) => void;
+  }): void {
```

Comment:
#q why not just make this function async rather than use callback pattern? 
```suggestion
public presentTradeExecutionSheet(input: {
     tradeQuote: TradeQuote;
     presentationBackground?: PresentationBackground;
}): Promise<({ kind: "success" } & Trade) | { kind: "failure"; error: string }>
```

Reply:
Talked about pro/cons.